docs/machine-learning/ml-tpm-guidance-blog.md

```diff
@@ -0,0 +1,99 @@
+
+### You need more people, like right now
+
+An ML Project has multiple stages, and each stage may require additional roles. For example, Design Research & Designers for Human Experience, Data Engineer for Data Collection, Feature Engineering, a Data Labeler for labeling structured data, engineers for MLOps and model deployment and the list can go on. As a TPM, you need to factor in having these resources available at the right time to avoid any schedule risks.
+
+### Your Feature is not my Feature
+
+TPM are familiar with Features, they are the core release mechanism for any project. However, in the ML world, Features have a whole different meaning. Feature Engineering enables the transformation of data so that it becomes usable for an algorithm. The input to Feature Engineering is raw data, and the output is generally called a Feature Vector. Creating the right features is an art and may require experimentation as well as domain expertise. Consider these factors when defining schedule and allocate time for domain experts in the project. For example, for a natural language processing engine for text extraction of financial documents, we hired financial researchers who were able to run a relevance judgment exercise with the engineering team to identify the right features.
+
+### It's a Biased world out there
+
+Bias in machine learning could be the number one issue of a model not performing to its intended needs. There are many reasons for bias to get introduced in the data which can impact the model performance. As a TPM, you can help your machine learning team by identifying the right use cases scenario and target personas. For example, for a person recognition algorithm, if the data source is only feeding a specific skin type, then production scenarios will not provide good results. Think about [Responsible AI principles](/docs/machine-learning/responsible-ai.md) from Day 1 to ensure fairness, security, privacy and transparency of the models. Shout out to [Bujuanes Livermore](https://www.linkedin.com/in/bujuaneslivermore/), Principal Design Researcher at Microsoft for relentlessly emphasizing on the Responsible AI principles across our CSE projects.
+
+## The learning Journey for a TPM
+
+As we can see, Machine Learning projects require the appreciation of few additional skills beyond the conventional software development processes. So, should you as a TPM, start learning about Gradient Descent and how to build Convolutional Neural Networks? Well, not exactly. (All power to you if you want to)
+
+The skills required for a TPM is to understand the nuances and complication of a Machine learning project, however it does not necessarily need a TPM to become a Data Scientist or an Applied ML Engineer. Here is a simple framework to think about your up-skilling for ML projects:
+
+![Skills for TPMs](/docs/machine-learning/images/ml-tpm-skills.png)
+
+### PM Fundamentals
```

Comment:
This is a good section to have in the playbook so I would add it to the new guidance file + I would also add the picture in this section too as I think it summarises the PM fundamentals well (in the blog post I would leave AS IS).

Suggestion there would be to have "Here are some references.." on a new line and add the resources as bullet points.

Reply:
moved to main md